openjdk/excludes/ProblemList_openjdk8-openj9.txt

```diff
@@ -170,6 +170,12 @@ sun/net/www/http/ChunkedOutputStream/checkError.java	https://github.com/adoptium
 
 ############################################################################
 
+# jdk_imageio
```

Comment:
There is an existing `jdk_imageio` section at L406 which can be combined with this.